web/src/App.css

```diff
@@ -1,32 +1,9 @@
-.App {
-  text-align: center;
+table {
+  width: 100%;
+  border-collapse: collapse;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+td {
+  border: 1px solid #cfcfcf;
+  padding: 0px 3px;
+}
```

Comment:
Please, for every file leave a `new line` at the end.

Reply:
@rianby64 ok, sorry, corrected it.